servers/republik/script/prolong/paperInvoice.js

```diff
@@ -77,12 +162,23 @@ PgDb.connect().then(async pgdb => {
       u.id != :PARKING_USER_ID
   `, { PARKING_USER_ID }))
     .filter(u => !benefactors.find(b => b.id === u.id))
-    .map(u => transformUser(u))
 
   log(`investigating ${paperPeople.length} paper people and ${benefactors.length} benefactors`)
 
+  const transaction = await pgdb.transactionBegin()
+
   const benefactorNeedProlong = await enrichWithProlongAndAddress(benefactors)
+    .then(createPledgeAndPayments.bind(
+      this,
+      { transaction, option: PROLONG_PACAKGEOPTION_BENEFACTOR }
+    ))
```

Comment:
I don't see the point with this bind construct. Why not just map to it?

Reply:
Yeah, I'd like to stick to it. Not really a mapping job here, more a contextual one: What transaction, what option to use.